blueprints/data-solutions/data-platform-foundations/demo/dataflow-csv2bq/README.md

```diff
@@ -3,7 +3,66 @@ This demo serves as a simple example of building and launching a Flex Template D
 
 ![Dataflow pipeline overview](../../images/df_demo_pipeline.png "Dataflow pipeline overview")
 
-## Example build run
+
+## Local development run
+
+For local development, the pipeline can be launched from the local machine for testing purposes using different runners depending on the scope of the test.
+
+### Using the Beam DirectRunner
+The below example uses the Beam DirectRunner. The use case for this runner is mainly for quick local run tests on low volume of data.
```

Comment:
I would mention that this would run the pipeline on the development environment.

Reply:
Done.